app/services/conversation_service/creator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ConversationService
+  class Creator
+    def initialize(sender:, receivers:, topic:)
+      @sender = sender
+      @receivers = receivers
+      @topic = topic
+      @conversations = []
+    end
+
+    def call
+      @receivers.each do |receiver|
+        @conversations << Conversation.create(sender: @sender, receiver: receiver, topic: @topic)
+      end
+      @conversations
```

Comment:
```suggestion
      @receivers.each do |receiver|
        receiver.conversations.create!(sender: @sender, receiver: receiver, topic: @topic)
      end
```
Always prefer using methods that raise exceptions, and then catch them depending on the case. That way, we have control over all cases. https://apidock.com/rails/v5.2.3/ActiveRecord/Persistence/ClassMethods/create%21